packages/common/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './llm/schemas'
```

Comment:
**tought:** If this is actually a common package, we should use strict namespaces. I was thinking of: 

```ts
// src/index.ts
export * as openai from './openai'
export * as sentry from './sentry' // this is currently in sdk-addons, but eventually here

// src/openai/index.ts
export * from './generate-content' // (rename this file)
export * as schemas from './schemas.ts'
```